### ISSUE TITLE
NoClassDefFoundError with FluidToFluid transform

### ISSUE THREAD

**Issue by pupnewfster** [xt9/InWorldCrafting#2]
JavaFX is not included so it is not able to find the class so nothing happens.

Version 0.11

Error that shows in CraftTweaker Log: https://pastebin.com/mmydfq56

Script

```
import mods.inworldcrafting.FluidToFluid;

FluidToFluid.transform(<liquid:lava>, <liquid:water>, <minecraft:blaze_powder>);
```

**Comment by xt9** [comment]
Thanks for sharing the log. I'll just switch to using a map.

**Comment by xt9** [comment]
Fixed in 481daffc3b0af57625d7b0fae20594084f832f66.
